apps/supernova/src/components/alerts/shared/AlertLinks.tsx

```diff
@@ -24,7 +24,11 @@ const AlertLinks = ({ alert, className }: any) => {
       {alert?.labels?.playbook && (
         <a
           className="underline"
-          href={`https://operations.global.cloud.sap/${alert?.labels?.playbook}`}
+          href={
+            alert.labels.playbook.startsWith("http")
```

Comment:
Please use optional chaining here to prevent potential runtime errors in cases when the `playbook` label doesn't exist. Like this:
```suggestion
            alert?.labels?.playbook?.startsWith("http")
```

Reply:
Actually I just saw that optional chaining is used at the beginning of the block, so this part shouldn't even be reached if `playbook` or another part of the chain is undefined. So all good, no need to do this.